CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+* @zarf-dev/maintainers @zarf-dev/reviewers
+
+/CODEOWNERS @zarf-dev/tsc
```

Comment:
we need to add these teams to the repo permissions.

Reply:
Will do!